app/helpers/ui_helper.rb

```diff
@@ -0,0 +1,5 @@
+module UiHelper
+  def return_to_dashboard_button
+    link_to 'Return to Dashboard', root_path, { class: "btn btn-info pull-right" }
+  end
+end
```

Comment:
Missing line ending, might want to make your editor not do this 
Next time we do a linting pass this will get updated, no worries